src/masternodeman.cpp

```diff
@@ -657,12 +657,6 @@ std::vector<std::pair<int64_t, MasternodeRef>> CMasternodeMan::GetMasternodeRank
 
 int CMasternodeMan::ProcessMNBroadcast(CNode* pfrom, CMasternodeBroadcast& mnb)
 {
-    // Skip after legacy obsolete. !TODO: remove when transition to DMN is complete
-    if (deterministicMNManager->LegacyMNObsolete()) {
-        LogPrint(BCLog::MASTERNODE, "mnb - skip obsolete message\n");
-        return 0;
-    }
-
```

Comment:
This is unreachable code, as we bail out early in `CMasternodeMan::ProcessMessageInner`, when legacy obsolete (since 70fe35071ffc4b57fd8f6083ff30dde1e39af72b, which is already in master).

I probably squashed the change inside the wrong commit, it was meant to be done in 4edf7aa0b4b181b7be75d044b63c3416f4852310. fixing.

Reply:
k thanks.

Reply:
yeah ok, great.